src/mockServer/msw/dexie-db.ts

```diff
@@ -41,6 +43,7 @@ export class AppMocks extends Dexie {
       appData: 'id, item.id, [item.id+creator.id], member.id, type, visibility',
       appSetting: 'id, item.id, name',
       appAction: 'id, memberId',
+      uploadedFiles: 'id',
```

Comment:
For this I think it would be better to create a migration, so that existing databases can be correctly migrated. Not strictly an issue, but I think it would be a good practice.

Here you should add it inside a `db.version(2).stores({...})` following [the docs](https://dexie.org/docs/Tutorial/Design#database-versioning)